tanner/emulators/php_object_injection.py

```diff
@@ -0,0 +1,51 @@
+import aiohttp
+import asyncio
+import logging
+
+from tanner import config
+from tanner.utils import patterns
+
+
+class PHPObjectInjection:
+    def __init__(self, loop=None):
+        self._loop = loop if loop is not None else asyncio.get_event_loop()
+        self.logger = logging.getLogger('tanner.php_object_injection')
+
+    async def get_injection_result(self, code):
+        object_injection_result = None
+
+        vul_code = "<?php " \
+                   "class ObjectInjection {" \
+                   "public $insert; " \
+                   "public function __destruct() " \
+                   "{ " \
+                   "$var = system($this->insert, $ret); " \
+                   "print $var[0]; }" \
+                   "} " \
+                   "$cmd = unserialize(\"%s\");" \
+                   "?>" % code
+
+        phpox_address = 'http://{host}:{port}'.format(host=config.TannerConfig.get('PHPOX', 'host'),
```

Comment:
you could create a helper function for this considering we are now using the PHP sandbox in multiple places.

Reply:
Great! @rnehra01 What do you think?

Reply:
Done.